package.json

```diff
@@ -4,18 +4,23 @@
   "version": "0.0.0",
   "description": "Minimalistic browser-based Solidity IDE",
   "scripts": {
-    "browser-test": "nightwatch --config nightwatch.js --env local",
-    "browser-test-remote-chrome": "nightwatch --config nightwatch.js --env chrome",
-    "browser-test-remote-firefox": "nightwatch --config nightwatch.js --env default",
-    "browser-test-remote-ie": "nightwatch --config nightwatch.js --env ie",
-    "browser-test-remote-safari": "nightwatch --config nightwatch.js --env safari",
-    "browser-test-remote-parallel": "nightwatch --config nightwatch.js --env safari,ie,default,chrome",
     "build": "mkdir build; browserify src/index.js -g yo-yoify -o build/app.js -t [ babelify ]; babel --plugins=transform-es2015-template-literals build/app.js --out-file build/app.js",
+    "test": "standard; babel src --out-dir babelify-src; node test/index.js",
+    "test-browser": "npm-run-all -lpr _selenium _http-serve browser-test",
+    "test-sauce": "env-cmd .env.local node sc",
     "csslint": "csslint --ignore=order-alphabetical --errors='errors,duplicate-properties,empty-rules' --exclude-list='assets/css/font-awesome.min.css' assets/css/",
-    "downloadsolc": "rm soljson.js; wget https://ethereum.github.io/solc-bin/soljson.js",
     "lint": "standard",
-    "serve": "http-server .",
-    "test": "standard; babel src --out-dir babelify-src; node test/index.js"
+    "browser-test": "nightwatch --config nightwatch.js --env local",
+    "browser-test-remote-chrome": "env-cmd .env.local nightwatch --config nightwatch.js --env chrome",
+    "browser-test-remote-firefox": "env-cmd .env.local nightwatch --config nightwatch.js --env default",
+    "browser-test-remote-ie": "env-cmd .env.local nightwatch --config nightwatch.js --env ie",
+    "browser-test-remote-safari": "env-cmd .env.local nightwatch --config nightwatch.js --env safari",
+    "browser-test-remote-parallel": "env-cmd .env.local nightwatch --config nightwatch.js --env safari,ie,default,chrome",
+    "_http-serve": "execr --silent http-server .",
```

Comment:
1) yes please change it back to `serve` . also because the remix repo also use `serve` (btw at some point we will need to push changes of this PR to the remix repo also :).
That 's be a good idea to improve our naming convention but I would rather do that in another process.
btw `serve` is not necessarily private because we often start a web server with `npm run serve` from the command line.

2) ok I see. in a way I agree with you... but if I run manually selenium or `http-server` from my local computer I'd like to see logs.
for instance `http-server` gives me the `userAgent` (that could be helpful), and selenium also logs interesting stuff.